pyproject.toml

```diff
@@ -35,18 +35,19 @@ documentation = "https://softwaremill.github.io/FoXAI/"
 "Bug Tracker" = "https://github.com/softwaremill/foxai/issues"
 
 [tool.poetry.dependencies]
-python = ">=3.7.2,<3.11"
+python = ">=3.8.0,<3.11"
 numpy = ">=1.21.4,<2.0.0"
 captum = ">=0.5.0,<1.0.0"
-torch = ">=1.12.1,<2.0.0"
-pytorch-lightning = ">=1.5.0,<2.0.0"
+torch = ">=2.0.0,<=2.0.1"
+pytorch-lightning = ">=2.0.0,<=2.0.4"
 wandb = ">=0.13.7,<1.0.0"
-pandas = ">=1.1.0,<2.0.0"
+pandas = ">=2.0.0,<=2.0.3"
```

Comment:
On the other hand, I am not sure if we are not cutting too many users with support only with ``pandas`` 2.0. PyTorch is actually core library for us so I think it would be easier to use version 2.0, but pandas I think has more of a supplementary role in our repo.

Reply:
Pandas is used in the library only in one place - in the CLI tool for updating experiments in W&B to access `DataFrame`. Which package has requirements for pandas `2.0.+`?

Reply:
outdated - resolving